app/assets/v2/js/pages/bounty_details.js

```diff
@@ -873,19 +873,35 @@ var do_actions = function(result) {
     actions.push(_entry);
   }
 
+  function get_repository_name(github_url) {
+    const repo_link = document.createElement('a');
+
+    repo_link.href = github_url;
+
+    const hostname = repo_link.hostname;
+
+    if (hostname.match('github')) {
```

Comment:
is this function pulling the repo name as the function name implies, or the hostname?